src/irr/video/COpenGLSpecializedShader.cpp

```diff
@@ -291,6 +288,9 @@ auto COpenGLSpecializedShader::compile(const COpenGLPipelineLayout* _layout) con
 	if (logbuf[0])
 		os::Printer::log(logbuf, ELL_ERROR);
 
+	if (m_locations.empty())
```

Comment:
what if there are no uniforms at all? will we keep on hitting this?

Reply:
ah nvm its in `compile` so only one hit ever.

Reply:
not ever, every time pipeline with this shader is created. Yes, we hit every time then. And then it runs 0 iterations loop so whatever

Reply:
yeah, I realized